org-formation/090-systems-manager/Scheduled-Script-Automation.yaml

```diff
@@ -277,7 +277,7 @@ Resources:
     Type: "AWS::SSM::Document"
     Properties:
       DocumentType: Automation
-      Name: "Automation-Run-Script"
+      Name: "Automation-Run-Script-v2"
```

Comment:
It's not required, could we just omit it?

Reply:
Maybe, but the document does show up in the AWS Console (under the SSM Documents section) and it would be nice for it to have a meaningful, human-readable name.

Reply:
I think that's what is does when not specified (name of the resource (possibly truncated) and some random string at the end). My only concern is if next time we modify it we forget to rename it.

Reply:
OK, I have removed the name.